sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityClient.java

```diff
@@ -23,38 +24,62 @@
 @ServiceClient(builder = CommunicationIdentityClientBuilder.class, isAsync = false)
 public final class CommunicationIdentityClient {
 
-    private final CommunicationIdentityImpl client;
+    private final CommunicationIdentitiesImpl client;
```

Comment:
Why is this plural?

Reply:
It is from generated code "sdk\communication\azure-communication-identity\src\main\java\com\azure\communication\identity\implementation\CommunicationIdentityClientImpl.java"